rust/parquet/src/errors.rs

```diff
@@ -33,7 +32,7 @@ quick_error! {
       /// Returned when code violates normal workflow of working with Parquet files.
       General(message: String) {
           display("Parquet error: {}", message)
-              description(message)
+              display(message)
```

Comment:
We already have a display statement? Why another one?

Reply:
My mistake, was doing this very late at night. I've removed the duplicates